src/main/com/example009/radek/v3/FileService.java

```diff
@@ -0,0 +1,93 @@
+package main.com.example009.radek.v3;
+
+import java.io.*;
+import java.util.List;
+import java.util.Scanner;
+
+public class FileService {
+
+    final private String path = "src/main/com/example009/radek/v3/calculator-results/";
+
+    void storeResultsIntoFile(List<Double> inputList) {
+
+        String name = getUserInput("Please insert file name: ", false);
+        saveResultsIntoFile(name, inputList);
+    }
+
+    void loadResultsFromFile() {
+
+        try {
+            String name = getUserInput("Please insert file name: ", true);
+
+            BufferedReader reader = new BufferedReader(new FileReader(path + name));
+            reader.lines().forEach(System.out::println);
+            reader.close();
+        } catch (FileNotFoundException e) {
+            System.out.println("Connecting to file reader wasn't successful");
+            e.printStackTrace();
+        } catch (IOException e) {
+            System.out.println("Reading from file wasn't successful");
+            e.printStackTrace();
+        }
+    }
+
+    String getUserInput(String message, boolean checkExistence) {
```

Comment:
ten flag checkExistence odstran a vyres to metodou, ktera se v pripade potreby bude volat, logicky to tu spravne nepatri 
kdysi jsem mel tendenci psat taky same flagy(boolean, co ti pomuuze rozhodnout), ale to signalizuje spatne napsany kod, takze zkusme se jim vyvarovat ... na projektu si ted nevybavim, kde bychom to meli, maximalne v setteru, kde nastavujes hodnotu u atributu, ale ne metodu s logikou

Reply:
Snad je to takto správně